assets/js/components/notifications/ErrorNotifications.test.js

```diff
@@ -90,4 +90,21 @@ describe( 'ErrorNotifications', () => {
 			} )
 		);
 	} );
+
+	it( 'renders GTE message for unsatisfied tagmanager readonly scope', () => {
```

Comment:
I'd suggest a tweak here:

```suggestion
	it( 'renders the GTE message when the only unsatisfied scope is the tagmanager readonly scope', () => {
```

Reply:
Additionally, it would be worth adding a test for the case where the scope exists but is not the only unsatisfied scope...